src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -251,11 +248,7 @@ public CompletableFuture<String> lookupAcquisitionUnitIdsByName(String data) {
 
   public CompletableFuture<Object> lookupAcquisitionUnitIdsByAccount(String data) {
```

Comment:
Please add java doc why this method just proxying, because it's not obvious approach

Reply:
Done